gnomad_qc/v5/sample_qc/hard_filters.py

```diff
@@ -59,6 +60,21 @@ def compute_aou_sample_qc(
         vds, intervals=telomeres_and_centromeres.ht(), keep=False
     )
 
+    logger.info("Excluding loci with more than 100 alternative alleles...")
+    # NOTE: Filtering loci with >100 alleles has no effect on the test dataset,
+    # since it contains no such loci. However, this filtering is still necessary
+    # on the full dataset because the number of alleles can change after removing
+    # samples from the raw VDS. As a result, it may not always match the 'EXCESS_ALLELES'
```

Comment:
can you expand on what `EXCESS_ALLELES` is here or add a link? it's good to assume whoever is reading this script in the future doesn't know or doesn't remember what that field is